question-servers/elements/pl_file_upload.py

```diff
@@ -0,0 +1,63 @@
+import lxml.html
+from html import escape
+import chevron
+import to_precision
+import prairielearn as pl
+import json
+
+def prepare(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    required_attribs = ["filenames"]
+    optional_attribs = ["answers_name"]
+    pl.check_attribs(element, required_attribs, optional_attribs)
+    name = pl.get_string_attrib(element, "answers_name", "_files")
+
+    return data
+
+def render(element_html, element_index, data):
```

Comment:
If there is already a submission (so `data["submitted_answers"][name]` exists) then we should probably restore it? At the moment, if you upload a file, click "Save", and then immediately click "Grade" then it will fail. Other elements (e.g., `pl_number_input`) restore the existing submitted answer so that you can see your current answer in the question.